pyfr/integrators/dual/pseudo/pseudocontrollers.py

```diff
@@ -121,6 +113,7 @@ def __init__(self, *args, **kwargs):
             'pyfr.integrators.dual.pseudo.kernels.localerrest'
         )
 
+        self.pintgkernels['localerrest'] = []
```

Comment:
Can we not make pintgkernels a regular dict(list)?

Reply:
declaring pintgkernels as a defaultdict(list) allows me to remove this line, but dict(list) gives an error